### PR TITLE
Refs #93. Now it's possible to modify descriptions

### PR DESCRIPTION
**Pull request by GibbOne**
Now it's possible to modify build configuration and project build descriptions.

According to:
- https://confluence.jetbrains.com/display/TW/REST+API#RESTAPI-ProjectSettings
- https://confluence.jetbrains.com/display/TW/REST+API#RESTAPI-BuildConfigurationAndTemplateSettings

### CODE REVIEW

**Comment by borismod** [comment]
@GibbOne it seems that the build failed:
https://ci.appveyor.com/project/QualiSystems/fluenttc-m35t3/build/1.0.24

**Comment by GibbOne** [comment]
@borismod it's due to the fact that I'm using a C# 6 feature, but the build agent uses MSBuild 12.0.
So "nameof" operator didn't exist. My mistake.
I can use simple constant and fix it.

But do you think to pass to C# 6 in the future?

**Comment by borismod** [comment]
@GibbOne thank you for your contribution.
Can you please add tests that cover the added functionality?

AcceptanceTests are a good place to look for examples:
https://github.com/QualiSystems/FluentTc/blob/master/FluentTc.Tests/AcceptanceTests.cs

In addition you can also add unit tests

File: FluentTc.Tests/AcceptanceTests.cs [hunk 1]
```diff
@@ -1163,5 +1163,131 @@ public void GetSingleSuccessfulFinishedBuild()
...
+        [TestCase(false)]
+        public void SetBuildDescription(bool paused)
+        {
+            // Arrange
+            var teamCityCaller = CreateTeamCityCaller();
+
+            var connectedTc = new RemoteTc().Connect(_ => _.AsGuest(), teamCityCaller);
+
+            // Act
+            connectedTc.SetBuildConfigurationField(_ => _.Id("BuildId"), __ => __.Paused(paused));
```
Comment (borismod) [inline FluentTc.Tests/AcceptanceTests.cs]: In terms of more readable API I would prefer.

```C# 
connectedTc.SetBuildConfigurationField(_ => _.Id("BuildId"), __ => __.Paused());
```
and 
```C#
connectedTc.SetBuildConfigurationField(_ => _.Id("BuildId"), __ => __.NotPaused());
```

The same for Archived

Thanks!

**Comment by borismod** [comment]
@GibbOne looks great! Thanks for your contribution. I am going to accept it. A new nuget package will be released soon.

**Comment by GibbOne** [comment]
Thank you!

**Pull request merged**